interface.html

```diff
@@ -439,13 +439,7 @@ <h4 class="panel-title" data-toggle="collapse" data-parent="#permissions-accordi
 
         <div class="user-datasource-options hidden">
           <div class="form-group clearfix">
-            <div class="col-xs-12 select-user-datasource-holder">
-              <label>Select the datasource where app users are listed <small>(Required)</small></label>
-              <p><small>To enable features that are specific to a user we need to know how to identify different users. Please select the location where the users emails are listed.</small></p>
-              <select name="select_user_datasource" id="select_user_datasource" class="form-control">
-                <option value="none">-- Select a data source</option>
-                <option disabled>------</option>
-              </select>
+            <div class="col-xs-12 select-user-datasource-holder ui-sortable" id="user_data_source_provider">
```

Comment:
Why does this need the `ui-sortable` class? How is this being sorted?

```suggestion
            <div class="col-xs-12 select-user-datasource-holder" id="user_data_source_provider">
```

Reply:
@tonytlwu it's needed for the data source provider so it would load the correct provider. It was suggested by @squallstar in this [comment](https://weboo.atlassian.net/browse/OD-7?focusedCommentId=10035) and we use it in every data source provider integration

Reply:
Seems very odd that `ui-sortable` is what does the job, but OK 👌🏻